### PR TITLE
Adds a function to export bookmarks.html file for Chrome

### PR DESCRIPTION
**Pull request by robmadden**
Playing around with a simple function to export a `bookmarks.html` file.

### CODE REVIEW

File: lib/scripts/bookmarks/index_test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    hears: (patterns, types, callback) => {
+      expect(patterns).toBe('(.*)bookmarks(.*)');
+      expect(types).toEqual(['direct_message', 'direct_mention', 'mention']);
+
+      const mockMessage = { cool: 'beans' };
+
+      callback({
+        reply: (message, text) => {
+          expect(message).toEqual(mockMessage);
+          expect(text).toMatch(/^<!DOCTYPE NETSCAPE-Bookmark-file-1>/);
```
Comment (maxbeatty) [inline lib/scripts/bookmarks/index_test.js]: is this a valid doctype from this century?

Comment (robmadden) [inline lib/scripts/bookmarks/index_test.js]: lol, that's what they have in their exported HTML file.

**Comment by robmadden** [comment]
The ides is that for anyone else who joins, they can query Data for a common set of bookmarks we all use.

**Comment by robmadden** [comment]
In other words, we don't have to constantly tell them where to go to look for stuff, they can do it themselves.

**Comment by maxbeatty** [comment]
ah I see. my only concern is keeping the bookmarks updated (branch, commit, PR, release). would the wiki be better for these? [custom slackbot responses](https://get.slack.help/hc/en-us/articles/202026038-Slackbot-personal-assistant-and-helpful-bot-)?

**Comment by robmadden** [comment]
Yeah...doesn't matter to me, wiki, slack, whatevs. Also, I realized that the bookmarks file attached has all of my personal bookmarks would encapsulates my Spaceback bookmarks 😉

**Pull request closed without merging**